esmvaltool_sample_data/loader.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+
+import cf_units
+import iris
+
+base_dir = Path(__file__).parent
+
+
+def strip_attributes(cube):
+    """Remove attributes that cause issues with merging and concatenation."""
+    for attr in ['creation_date', 'tracking_id', 'history']:
+        if attr in cube.attributes:
+            cube.attributes.pop(attr)
+
+
+def simplify_time(cube):
+    coord = cube.coord('time')
+    coord.convert_units(
+        cf_units.Unit('days since 1850-1-1 00:00:00',
+                      calendar=coord.units.calendar))
+
+
+def load_cubes_from_input_dirs(input_dirs):
+    """Loads all *.nc files from each input dir into a cube."""
+    for input_dir in input_dirs:
+        files = input_dir.glob('*.nc')
+        cubes = iris.load(str(file) for file in files)
+        for cube in cubes:
+            strip_attributes(cube)
+            simplify_time(cube)
+
+        cubes = cubes.concatenate()
+        cube = cubes[0]
```

Comment:
```suggestion
        cube = cubes.concatenate_cube()
```
try if this works, that will allow only a single resulting cube. If it doesn't work because of the data, we can go back to the old version (or think of something slighly smarter if needed)

Reply:
Works for all but 1: `esmvaltool_sample_data/data/timeseries/CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Amon/ta/gn/v20190914`

Reply:
It looks like that historical experiment mistakenly contains data from the future in an older version (v20190914):
```
CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Amon/ta/gn/v20190914/ta_Amon_NorCPM1_historical_r1i1p1f1_gn_185001-201412.nc
CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Amon/ta/gn/v20190914/ta_Amon_NorCPM1_historical_r1i1p1f1_gn_201501-201812.nc
CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Amon/ta/gn/v20190914/ta_Amon_NorCPM1_historical_r1i1p1f1_gn_201901-202912.nc
```
the latest version (v20200724) looks fine:
```
CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Amon/ta/gn/v20200724/ta_Amon_NorCPM1_historical_r1i1p1f1_gn_185001-201412.nc
```
so if we could solve #5 that would also solve this